appdata-release-tag.patch

```diff
@@ -0,0 +1,12 @@
+--- a/data/org.gnome.Sudoku.appdata.xml.in	2019-06-04 14:48:52.126188662 +0200
++++ b/data/org.gnome.Sudoku.appdata.xml.in	2019-06-04 14:47:34.211892440 +0200
+@@ -46,6 +46,9 @@
+     <kudo>ModernToolkit</kudo>
+     <kudo>UserDocs</kudo>
+   </kudos>
++  <releases>
++    <release version="3.32.0" date="2O19-03-11" />
```

Comment:
This might be hard to spot, but it actually says `2O19` with capital O, not zero.